cfme/automate/service_dialogs.py

```diff
@@ -73,17 +73,16 @@ def is_displayed(self):
         )
 
 
-class DialogCollection(Navigatable):
+class DialogCollection(NavigatableMixin):
```

Comment:
please make sure, that this is fixed in another PR

Reply:
I is already in another pr #5396